tests/reg.spec.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import Jest from 'jest';
+import toJson from 'enzyme-to-json';
+import { shallow } from './enzyme';
+import Registration from '../src/components/registration/index';
+
+
+describe('Application test', () => {
+  it('should work fine on App', () => {
+    const wrapper = shallow(<Registration />);
```

Comment:
Good job @olorunwalawrence , can we change this to `mount`? Use `shallow` when you are rendering a React component within the component that you are shallowing